src/app/manage-users/manage-user-updation/manage-user-updation.component.html

```diff
@@ -0,0 +1,4 @@
+<app-main-form 
+    [currentUser]="currentUser" 
+    (valuesSubmitted)="updateUser($event)">
```

Comment:
See my comment for `manage-user-creation` html

Reply:
done this